### ISSUE TITLE
UI: Separate Resources Under 'Videos' in 'Introduction to Open Source' Page

### ISSUE THREAD

**Issue by Sriparno08** [Sriparno08/Openpedia#349]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [ ] Codebase
- [x] User Interface
- [ ] Feature Request

## Description

We need to separate the resources under 'Videos' in 'Introduction to Open Source' page by making them exist in separate boxes. It can look like this:

<p align="center">
  <img src="https://github.com/Sriparno08/Openpedia/assets/89148144/3df0d084-9ec2-4723-a6a4-93abf4d39c10">
</p>

**Comment by Bibek478** [comment]
Would you please assign me this issue under jwoc24

**Comment by Sriparno08** [comment]
You're already assigned to another issue. Once your PR for it is merged, you will be considered for this issue. If anyone asks to be assigned before your PR is merged, he/she will be assigned.

**Comment by Bishal-35** [comment]
I would like to work on it under JWoC'24

**Comment by Sriparno08** [comment]
@Bishal-35 You're assigned.

**Comment by Bishal-35** [comment]
can you explain it abit?

**Comment by Sriparno08** [comment]
We want to make it look like this:

![image](https://github.com/Sriparno08/Openpedia/assets/89148144/6f76d77b-5a65-4594-8036-fce6913920f4)

Suggestions:

1. Remove the 'Videos' header and the background color on which the resources exist.
2. Put each resource in a separate box and place the boxes sequentially, just as the image suggests.

**Comment by Bishal-35** [comment]
Okay
